README.md

```diff
@@ -118,6 +118,19 @@ Ongoing run identifiers to define `RUN_ID` can be retrieved by listing the ongoi
 
 You can monitor ongoing run using [mlflow](https://mlflow.org). By default a local instance of mlflow is started by cogment-verse and is accessible at <http://localhost:3000>.
 
+#### StableBaselines3 - HuggingFace Hub
```

Comment:
Can you move that to the documentation, e.g. in docs/integration/huggingface_sb3.md